packages/sdks/snippets/sveltekit/src/routes/live-preview/+page.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import type { BuilderContent } from '@builder.io/sdk-svelte';
+  import { onMount, onDestroy } from 'svelte';
+  import { subscribeToEditor } from '@builder.io/sdk-svelte';
+
+  export let data: {
+    content: BuilderContent | null;
+  };
+
+  let content: BuilderContent | null = data.content;
+  let isLoading = !content;
+
+  let unsubscribe: (() => void) | null = null;
+
+  onMount(() => {
+    unsubscribe = subscribeToEditor({
+      model: 'blog-data',
+      apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
+      callback(updatedContent) {
+        content = updatedContent;
```

Comment:
instead of having `data.content` assigned to `content`, do you know if its possible to reassign values to `data.content` directly and remove the need for the `content` variable?

by that i mean:
```suggestion
        data.content = updatedContent;
```

does that work in svelte/sveltekit? I'm not 100% sure but would simplify this code a bit